pinot-core/src/main/java/org/apache/pinot/core/query/reduce/HavingFilterHandler.java

```diff
@@ -154,6 +154,7 @@ public boolean isMatch(Object[] row) {
         case TIMESTAMP:
           return _predicateEvaluator.applySV(((Timestamp) value).getTime());
         case STRING:
+        case JSON:
```

Comment:
We won't reach here as `JSON` is not supported for predicate evaluator yet

Reply:
Removed.